handbook/engineering/campaigns/goals.md

```diff
@@ -1,66 +1,65 @@
-# Campaigns goals and priorities
+# Campaigns goals
 
-## Goals
+## Design and build the best experience for creating changesets across many repositories
 
-### Build a base of 10 customers who have created 80 campaigns and made 4000 changesets in Q4
+Campaigns need to be easy to explain, easy to understand, easy to get started, and easy to explore.
 
-We want to get a solid set of customers who regularly use and rely on campaigns. 
+How do we do this:
+  * Clean conceptual model
+  * Excellent docs, tutorials, demos/training
+  * Expressive syntax
+  * Best debugging experience
 
-What will get us there?
+Our success metrics:
+  * TBD
 
-- **Proactive outreach** Campaigns is new and transformative software. Many companies are not yet aware they need large scale code modifications. We need to find those companies within our customer base and introduce them to campaigns. This process will also teach us about marketing campaigns.
-- **Smoother on-ramp:** Internal testing has revealed that using campaigns for the first time is not as easy as it could be. A frictionless initial experience is critical to adoption and a delightful user experience.
-- **Better debugging experience:** It can be challenging to debug a more complex campaign. Figuring out what went wrong and how to fix it is a slow iterative loop.
-- **User validation:** The above issues were revealed by user testing. Once we believe we have fixed them, we need to validate this with further user testing.
+## Build the best solution for managing large numbers of changesets
 
-### Grow adoption of campaigns
+Much of the work in completing a large scale change across many repos is in getting the changesets merged. We want to be the best tool for all aspects of changeset management.
 
-Once we’re out of beta, we want everyone to start using campaigns!
+How do we do this:
+  * Help users understand the overall status of their campaign
+  * Provide tools to find and resolve problematic changesets
+  * Provide flexible tools to prompt reviewers and owners to merge the created changesets in whatever way best suits the customer
+  * Integrate with issue tracking systems
```

Comment:
This actually feels like it fits more under the next _Increase the scope of campaigns_ heading to me.

Reply:
It increases the scope of our work, but what we meant by "scope" was "the number of situations in which it makes sense to run campaigns". (If you have a better word than "scope", please suggest it. "Scope of potential usage" maybe?)

So given that, I feel like integrating with issue tracking doesn't increase that kind of scope. It does make it easier to manage a campaign at a company where everything is tracked through an issue tracking system, and would help the stated success metric of reducing seconds from changeset created to changeset merged.